Detectors/MUON/MCH/Base/src/Digit.cxx

```diff
@@ -29,7 +35,7 @@ bool Digit::operator==(const Digit& other) const
   return mDetID == other.mDetID &&
          mPadID == other.mPadID &&
          mADC == other.mADC &&
-         closeEnough(mTime, other.mTime);
+         closeEnough(mTime.sampaTime, other.mTime.sampaTime);
```

Comment:
with a time that's now an int, closeEnough is no longer needed

Reply:
Ok, removed.